.github/workflows/ci-integration.yaml

```diff
@@ -84,7 +84,7 @@ jobs:
         if: matrix.platform == 'lxd' || matrix.platform == 'vm'
         run: sudo lxd init --auto
       - name: Git checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: Retieve debs
```

Comment:
I know it is still a WIP, but let me request a tiny change. Please change to `Retrieve`.

Reply:
Done.